### PR TITLE
OCP 4.13 - IBM Power Virtual Server using installer-provisioned infrastructure follow up

### PR DESCRIPTION
**Pull request by mburke5678**
Follow up to https://github.com/openshift/openshift-docs/pull/47558

openshift-4.13+

Addresses: 
* https://github.com/openshift/openshift-docs/pull/47558#discussion_r1194277484
* https://github.com/openshift/openshift-docs/pull/47558#discussion_r1194276007

### CODE REVIEW

**Comment by mburke5678** [comment]
@alishaIBM PTAL

File: modules/installation-uninstall-clouds.adoc [hunk 1]
```diff
@@ -128,7 +128,7 @@ directory to delete the cluster.
 ifdef::ibm-power-vs[]
 [NOTE]
```
Comment (kalexand-rh) [inline modules/installation-uninstall-clouds.adoc:129]:
```suggestion
+
[NOTE]
```

Comment (kalexand-rh) [reply modules/installation-uninstall-clouds.adoc:129]: Without this `+`, the numbering in the module breaks.

**Pull request merged**